src/Compatibility/Microsoft.DotNet.PackageValidation/Package.cs

```diff
@@ -0,0 +1,178 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using NuGet.Client;
+using NuGet.Common;
+using NuGet.ContentModel;
+using NuGet.Frameworks;
+using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.RuntimeModel;
+using NuGet.Versioning;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    public class Package
+    {
+        private ManagedCodeConventions _conventions;
+        private ContentItemCollection _packageAssets = new ContentItemCollection();
+        private string _packagePath;
+        private const string _nugetFeed = "https://api.nuget.org/v3/index.json";
```

Comment:
Do we have to hard-code this feed here? Would it be better if we could use the current NuGet context to get the current configured feed(s)?